mmv1/third_party/tgc/compute_instance_group.go

```diff
@@ -0,0 +1,155 @@
+package google
+
+import (
+	"reflect"
+
+	"github.com/GoogleCloudPlatform/terraform-google-conversion/v5/tfplan2cai/converters/google/resources/cai"
+	"github.com/hashicorp/terraform-provider-google-beta/google-beta/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google-beta/google-beta/transport"
+)
+
+const ComputeInstanceGroupAssetType string = "compute.googleapis.com/InstanceGroup"
+
+func resourceComputeInstanceGroup() cai.ResourceConverter {
```

Comment:
Nit: I think convention is to include "converter", ie. `resourceConverterComputeInstanceGroup`

Reply:
done, thanks